rules/rules-to-better-automation/rule.md

```diff
@@ -0,0 +1,10 @@
+---
+type: rule
+title: Rules to better automation
+uri: rules-to-better-automation
+authors:
+  - title: Jack
+created: 2023-12-07T05:38:25.785Z
+guid: 9557813f-a26f-4f42-a420-6031f0f5f78f
+---
+A social group's success often depends on its conventions—shared norms and agreements that guide communication, decision-making, and unity. Strong conventions help navigate challenges, resolve conflicts, and maintain a positive atmosphere, shaping the group's overall success. It's crucial to note that while automation introduces efficiency, it should not disrupt these conventions or alter the fundamental ways people interact and collaborate. Automation should complement and enhance established norms rather than break them, ensuring a seamless integration that preserves the group's cohesiveness.
```

Comment:
We do this all the time - we introduce automation that changes the way people work in the short term but gives efficiency benefits in the long term once people adjust.

If you want to make this rule you need to define 'convention' and at which points it's acceptable and unacceptable to break conventions.

It needs to be specific i.e. make it 'followable'